src/components/displaysettings/displaysettings.js

```diff
@@ -186,6 +186,8 @@ define(['require', 'browser', 'layoutManager', 'appSettings', 'pluginManager', '
         context.querySelector('#selectLanguage').value = userSettings.language() || '';
         context.querySelector('.selectDateTimeLocale').value = userSettings.dateTimeLocale() || '';
 
+        context.querySelector('#txtLibraryPageSize').value = userSettings.libraryPageSize() || 100;
```

Comment:
if `PageSize` is `0` then `100`.

Reply:
Maybe let `userSettings` return the correct value?
```suggestion
        context.querySelector('#txtLibraryPageSize').value = userSettings.libraryPageSize();
```

Reply:
@MrTimscampi Does it restore zero value when you reopen settings?